draft-song-lake-ra.md

```diff
@@ -98,277 +99,97 @@ How the components are separated between the secure and non-secure worlds on a d
+## Attestation Models {#models}
 
-### Background-check model -- Passport model {#mutual-attestation-BP}
+This specification supports both the RATS background-check model (see in {{bg}}) and passport model (see in{{pp}}).
+The corresponding EAD items for background-check model and the passport model are independent between each other.
+The detailed EAD items definitions are in {{ead-bg}} for the background-check model and {{ead-pp}} for the passport model.
 
-This use case is applicable when the EDHOC Initiator is constrained in terms of connectivity.
-In this case, the mutual attestation begins with a forward background-check attestation, followed by a reverse passport attestation.
+### Background-check Model (BG) {#bg}
 
-1. EDHOC Initiator sends the first EDHOC message with an Attestation proposal in EAD_1.
+In background-check model, the Attester sends the evidence to the Relying Party.
+The Relying Party transfers the evidence to the Verifier and gets back the attestation result from the Verifier.
 
-2. In EAD_2, two EAD items are included: an Attestation request responding to the EDHOC Initiator's proposal and a Result proposal to initiate a reverse passport attestation.
-The EDHOC Responder SHOULD connect to a Verifier to select the evidence type for the forward attestation in background-check model.
+An EDHOC session is established between the Attester and the Relying Party.
+A negotiation of the evidence type is required before the Attester sends the evidence.
+All three parties must agree on a selected evidence type.
 
-3. In EAD_3, two EAD items are included: an Evidence and a Result request.
-The EDHOC Responder SHOULD connect to the Verfier in the passport model(VP) indicated in the Result request to obtain the Result from that Verifier.
-Simultaneously, the EDHOC Responder sends the received Evidence to the Verifier in the background-check model(VB) to get the attestation result of the first forward attestation.
+The Attester first sends a list of the provided evidence types, formatted as an Attestation proposal in an EDHOC EAD item, to the Relying Party.
+The Relying Party transfers the list to the Verifier and should receive at least one supported evidence type from the Verifier.
+If the Relying Party receives more than one evidence type, a single evidence type should be selected by the Relying Party and sends it back as an Attestation request to the Attester.
 
-4. The EDHOC Responder sends the Result that is obtained from VP to the EDHOC Initiator in EAD_4, and consumes the Attetation result from VB.
+We use nonce (at least 8 bytes according to {{I-D.ietf-rats-eat}}) to guarantee the freshness for the attestation session, which is generated by the Verifier and sends to the Relying Party.
+The Relying Party puts nonce and the selected evidence type together in a tuple to generate an Attestation request as an EAD item.
```

Comment:
s/puts nonce/puts the nonce/

Reply:
I suggest you pass the text through a grammar corrector (can be ChatGPT) to identify small items like this and verbal tenses.